github/GithubRetry.py

```diff
@@ -112,7 +112,19 @@ def increment(
                         content = self.get_content(response, url)
                         content = json.loads(content)
                         message = content.get("message")
+                    except Exception as e:
+                        # we want to fall back to the actual github exception (probably a rate limit error)
+                        # but provide some context why we could not deal with it without another exception
+                        try:
+                            raise RuntimeError(
+                                "Failed to inspect response message"
+                            ) from e
+                        except RuntimeError as e:
```

Comment:
Maybe I miss something, but why raise an exception that you are also immediately absorbing so does not go anywhere...

Reply:
To create an exception chain, GithubException caused by RuntimeError, caused by e.